src/cassandra/cluster.rs

```diff
@@ -577,7 +577,7 @@ impl Cluster {
     /// Default: 30 seconds
     pub fn set_connection_heartbeat_interval(&mut self, hearbeat: Duration) -> &mut Self {
         unsafe {
-            cass_cluster_set_connection_heartbeat_interval(self.0, hearbeat.whole_seconds() as u32);
+            cass_cluster_set_connection_heartbeat_interval(self.0, hearbeat.as_secs() as u32);
```

Comment:
Let's fix typo `hearbeat` to `heartbeat` while we're here.

Reply:
Whoops, clicked merge >:( I will fix this in another PR.